categories.php

```diff
@@ -0,0 +1,19 @@
+<?php
+require_once ('vendor/autoload.php');
+header('Content-Type: application/json; charset=utf-8');
+header("Access-Control-Allow-Origin: *");
+use \FurnitureStoreApi\Categories\CategoryHydrator;
+
+try
+{
+    $results = ['message'=>'Successfully retrieved products',
+        'data' => CategoryHydrator::getCategories()];
+    echo json_encode($results);
+    http_response_code(200);
```

Comment:
Putting the results into `['message'=>, 'data'=>]` format is something that will happen quite often, so it's worth creating a `ResponseService` with a static method to take a message string and json string, and create a response from them.

Reply:
It should also take a number for the response code and set that.